client/layout/index.jsx

```diff
@@ -100,6 +101,20 @@ class Layout extends Component {
 		// intentionally don't remove these in unmount
 	}
 
+	renderMasterBar() {
```

Comment:
Nit: AFAIK `masterbar` is a single word in our a8c vocabulary, so this should be `renderMasterbar`.

Reply:
:+1: